src/draw/ElkLayout.js

```diff
@@ -126,6 +131,15 @@ class ElkLayout extends DefaultLayout {
 
       ...elkParams,
     };
+
+    /**
+     Parameters for the algorithm that places a single new component.
+     */
+    this.singleComponentParams = {
+      precision: 10,
```

Comment:
put precision and margin in configuration

Reply:
done